crates/mpc-attestation/build.rs

```diff
@@ -0,0 +1,187 @@
+use std::env;
+use std::fs::{self, File};
+use std::io::Write;
+use std::path::Path;
+use std::path::PathBuf;
+
+const ASSETS_DIR_NAME: &str = "assets";
+
+fn main() {
+    // Find assets directory
+    let manifest_dir =
+        env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR must be set by Cargo");
+
+    let assets_dir = PathBuf::from(&manifest_dir).join(ASSETS_DIR_NAME);
+
+    println!("cargo:rerun-if-changed={}", ASSETS_DIR_NAME);
```

Comment:
```suggestion
    println!("cargo::rerun-if-changed={}", ASSETS_DIR_NAME);
```

https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed